ibis/pandas/udf.py

```diff
@@ -216,30 +216,47 @@ def pre_execute_analytic_and_reduction_udf(op, *clients, scope=None, **kwargs):
     input_type = op.input_type
     nargs = len(input_type)
 
-    # An execution rule for a simple aggregate node
+    # An execution rule for ungrouped aggregations. Handles:
+    #   1) AggregationNode with no grouping
+    #   2) Some aggregation over a window with no grouping
     @execute_node.register(type(op), *(itertools.repeat(pd.Series, nargs)))
     def execute_udaf_node(op, *args, **kwargs):
```

Comment:
Perhaps update the name of this to `execute_udaf_node_no_grouping` to clearer

Reply:
or `execute_udaf_node_ungrouped` or that's the naming conversion you are using

Reply:
Done